cv_next/components/layout/popup.tsx

```diff
@@ -0,0 +1,40 @@
+"use client"
+
+import Image from "next/image";
+import settignsIcon from "../../public/images/closeIcon.png"
+import Link from "next/link";
+
+const navLinks = [
+  {
+    route: "Login",
+    path: "/login",
+  },
+  {
+    route: "Signout",
+    path: "/logout",
+  },
+]
+
+interface PopupProps {
+  closeCb: () => void;
+}
+
+export default function Popup({ closeCb }: PopupProps) {
+  return <div className="fixed top-0 w-full left-0 h-full flex justify-center items-center">
```

Comment:
Why not use HTML's dialog? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog
Also looks like there's no click-outside-to-close functionality, see a solution reference for example here: https://stackoverflow.com/a/57463812

Reply:
Switch to the HTML dialog and added the click outside functionality